### PR TITLE
fix: addon table style bug

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
fix addon table style bug

## Does this PR introduce a user interface change?
- [x] Yes(screenshot is required)
- [ ] No
image

## Which versions should be patched?
release/1.1

## Which issue(s) this PR fixes:

Fixes # addon table style bug

### CODE REVIEW

**Review by daskyrk — APPROVED** [review]

File: shell/app/modules/cmp/pages/middleware-dashboard/index.tsx [hunk 1]
```diff
@@ -153,15 +153,15 @@ const MiddlewareDashboard = () => {
       title: i18n.t('node'),
       dataIndex: 'nodes',
       key: 'nodes',
-      width: '10%',
+      width: 80,
```
Comment (daskyrk) [inline shell/app/modules/cmp/pages/middleware-dashboard/index.tsx:156]: Is it wide enough in English mode？

Comment (hujiahao-hjh) [reply shell/app/modules/cmp/pages/middleware-dashboard/index.tsx:156]: It's enough to 'node' column, But the 'number of references' column is not wide enough.

**Pull request merged**